protofire/gnosis-conditional-exchange
# Realitio proxy

This PR:

- Adds Realitio proxy contract to Docker
- Updates addresses
- Makes Realitio's timeout configurable via an environment variable

## How to test this

1. Set the timeout to some low value, like 10.
1. Create a market. Set the opening date at the current timestamp.
1. Get the questionId of the market.
1. Interact with the Realitio contract. Call the `submitAnswer` method (e.g. `submitAnswer(questionId, 1)`).
1. Wait 10 seconds.
1. Make sure a new block is mined in ganache (you can use eth-cli's `eth tx:nop` for example).
1. Interact with the RealitioProxy contract. Call the `resolveCondition` method with the `questionId`.
1. The condition should be resolved. If you used `1` in step 4, then the second answer should be marked as the winning one.
